src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -511,6 +512,7 @@ private void commitOffset() throws DeltaFailureException {
     try {
       Failsafe.with(commitRetryPolicy).run(() -> {
         if (latestOffset != null) {
+          LOG.debug("Committing offset : {} and seq num: {}" , latestOffset.get(), latestSequenceNum);
```

Comment:
nit: remove these debug logs.

Reply:
as discussed offline, change it to trace level.